src/server/hook_func.c

```diff
@@ -153,11 +153,11 @@ extern	char server_host[PBS_MAXHOSTNAME+1];
 /* Local Private Functions */
 
 /* Global Data items */
+int	do_sync_mom_hookfiles = 1;
```

Comment:
I think the variable name is self-documentable to indicate the need to sync mom hook files.